test/internals/reply.test.js

```diff
@@ -46,6 +46,14 @@ test('reply.serializer should set a custom serializer', t => {
   t.equal(reply._serializer, 'serializer')
 })
 
+test('reply.serialize should serialize payload', t => {
+  t.plan(1)
+  const response = { statusCode: 200 }
+  const context = { responseSchema: {} }
```

Comment:
I think this `responseSchema` should be specified a bit more.

Reply:
Also, `responseSchema` is actually as symbol key, not a string key, so it would need to be imported from the validation file:
https://github.com/fastify/fastify/blob/c759af7be4e720bc4fcae3b0c936b938645ddb7e/lib/validation.js#L127

Then used like this:

```js
const context = { [responseSchema]: { /* ... */ } }
```

Reply:
In general what is a ‘Symbol key’ how does it differ from a ‘string key’?